components/Tabs/Tabs.js

```diff
@@ -69,4 +88,8 @@ class TabCard {
 - In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
 
 */
-let tabs = document.querySelectorAll();
\ No newline at end of file
+let tabs = document.querySelectorAll(".tab").forEach(tab => new TabLink(tab));
+// console.log(tabs);
+// tabs.forEach(function (tab) {
+//   return new TabLink (tab);
+// });
```

Comment:
I know that I told you to change this, but it was actually right both ways. The version you ended up with is  a little more efficient, though :)